### ISSUE TITLE
Gephi 0.9.5 not responding after launch - macOS Monterey 12.3.1

### ISSUE THREAD

**Issue by TeodoroCriscione** [gephi/gephi#2521]
## Current Behavior

Hi,

I tried to install several versions of Gephi 0.9.X, but they do not respond after launching them. They all freeze at the initial step (see image below). I used older versions of Gephi before on the same laptop, but now something is not working.

<img width="1297" alt="Screenshot 2022-05-06 at 08 29 41" src="https://user-images.githubusercontent.com/44541241/167079347-081321e1-9802-49dd-8183-c6691646ee8f.png">

Thank you for your help and suggestions!

## Possible Solution

Compatibility issues with the current version of Gephi and macOS?

## Steps to Reproduce

1.  Install Gephi 0.9.5 on macOS Monterey 12.3.1

* Version used: Gephi 0.9.5
* Operating System: macOS Monterey 12.3.1

**Comment by mbastian** [comment]
Hi @TeodoroCriscione, we've definitely have Gephi working on Monterey 12.3.1 so I'm not sure what might be wrong with yours. Would you mind attaching your `messages.log` file situated at `/Users/USERNAME/Library/Application Support/gephi/0.9/var/log` so that I could inspect the logs? Thank you!

**Comment by tbuechner** [comment]
I am having the same problem, here is my `messages.log` file: [messages.log](https://github.com/gephi/gephi/files/8652397/messages.log)

**Comment by tbuechner** [comment]
For me, this fixed the problem: https://github.com/gephi/gephi/issues/1851#issuecomment-468584185

**Comment by mbastian** [comment]
@tbuechner Interesting, thanks for sharing. This additional parameter would disable the autoupdate check that happens at startup. Your log file doesn't show any error so it's still unclear to me what's going on. Are you behind a proxy or VPN? Does this issue manifest itself on different networks? Then, I also see that you have configured your own Java JRE. While it's still the same version we recommend (11) I'm curious if the issue persists with the default (embedded JRE). Thank you!

**Comment by tbuechner** [comment]
@mbastian You are right, this setting seems not to be related to the actual problem - I am not behind a proxy or VPN. And it is not related to the JRE, I switched back to the embedded JRE. It was just a coincidence that it worked with this setting, now it is unfortunately not working again.

Is there a debug mode or anything else I can do to provide you with more log statements?

**Comment by msancor** [comment]
This is happening to me too, I'm using gephi 0.9.5 and MacOS Monterey 12.4. Is there a solution yet?

**Comment by mbastian** [comment]
Hi @msancor, as you can see the comments so far didn't lead to understanding what might be happening. If you could post your `messages.log` file as well it would be helpful. Thank you.

**Comment by jeremyreffin** [comment]
I have posted a workaround that fixed a variation on this theme in #2546 

tl;dr I deleted the user config files and it then ran fine

In that version of the problem, the smaller Welcome hadn't yet rendered so may not be the same bug
